src/image-button/image-button.style.tsx

```diff
@@ -21,29 +21,31 @@ export const Main = styled.button`
 
     img {
         width: 3.5rem;
-    }
-
-    ${MediaQuery.MaxWidth.tablet} {
-        padding: 1.5rem 0rem;
+        height: 3.5rem;
     }
 
     ${MediaQuery.MaxWidth.mobileL} {
-        padding: 1rem 0rem;
+        padding: 0.5rem 0.25rem;
     }
 
-    ${MediaQuery.MinWidth.tablet} {
-        :focus:active {
-            box-shadow: ${DesignToken.ElevationBoxShadow};
-            ${TextStyleHelper.getTextStyle("H4", "bold")}
-        }
-
-        @media (hover: hover) {
-            &:hover {
-                box-shadow: 0px 8px 16px -8px ${Color.Shadow.Elevation};
-                ${TextStyleHelper.getTextStyle("H4", "bold")}
-            }
-        }
+    &:hover {
+        border: 1px solid ${Color.Accent.Light[1]};
+        box-shadow: ${DesignToken.InputBoxShadow};
```

Comment:
input box shadows are inset, follow Toggle's instead

Reply:
sure, Padding between text and image should be 4px for mobile, there's no selected + disabled state.